### PR TITLE
Fix license badge links in packages

### PR DESCRIPTION
**Pull request by jpeyper**
I noticed that if you clicked the license badges they were not pathing to the license at the root

### CODE REVIEW

**Comment by jpeyper** [comment]
@vivian-farrell @mpeyper when we publish the npm packages, should the license be copied in?

File: packages/redux-subspace/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![npm version](https://img.shields.io/npm/v/redux-subspace.svg?style=flat-square)](https://www.npmjs.com/package/redux-subspace)
 [![npm downloads](https://img.shields.io/npm/dm/redux-subspace.svg?style=flat-square)](https://www.npmjs.com/package/redux-subspace)
-[![License: MIT](https://img.shields.io/npm/l/redux-subspace.svg?style=flat-square)](LICENSE)
+[![License: MIT](https://img.shields.io/npm/l/redux-subspace.svg?style=flat-square)](/LICENSE.md)
```
Comment (jpeyper) [inline packages/redux-subspace/README.md]: this was just wrong before.

**Comment by mpeyper** [comment]
I'm happy with this.  @vivian-farrell will be best to answer if the license file needs to be in the published package or if the license field in the package.json and the the link to the one in the root of the repo is enough.

**Comment by jpeyper** [comment]
Lets merge this now, and if @vivian-farrell gets back to us I'll do another fix to do whatever is required.

**Comment by mpeyper** [comment]
I guess it less incorrect now than before.  Still might be incorrect.

**Pull request merged**